test/tedious/tedious.js

```diff
@@ -13,8 +13,10 @@ if (parseInt(process.version.match(/^v(\d+)\./)[1]) > 0) {
   require('../common/templatestring.js')
 }
 
+function clone (val) { return Object.assign({}, val) }
+
 const config = function () {
-  let cfg = JSON.parse(require('fs').readFileSync(`${__dirname}/../.mssql.json`))
+  let cfg = clone(require('../mssql-config'))
```

Comment:
I'd be in favour of assigning the mssql-config require once at the top (eslint `global-require` style) and then cloning that assignment like so:

```js
const configOptions = require('../mssql-config');

// later...

function config() {
    return Object.assign({ driver: 'tedious' }, configOptions);
}
```

Reply:
PRS WELCOME

Reply:
yup, that would be more readable